### ISSUE TITLE
Mapping parts to staves

### ISSUE THREAD

**Issue by clnoel** [w3c/mnx#97]
I mentioned this in #57, but I think it needs its own issue, so we can talk about it separately.

In many pieces, the mapping of parts to stavesis very straight forward. One staff (or geand staff) = One part.

However, there are cases when this is not true. Usually, the goal was probably to save paper, but in some cases I'm sure it actually improves readability. In particular, I recall a number of instances from my past in a SATB choir where my alto part was sometimes written in unison with sopranos, written as a chord with the soprano part, written downstem against the sopranos' upstem part, or written on its own staff. Sometimes all four modes would be used in the same piece. 

Conceptually, I would consider my alto part to be a `<part>` and its location on the staves to be a layout decision, but I'm not sure how we would represent that. It's also possible that we need to get further on #57 before we can take up this issue!

**Comment by joeberkovitz** [comment]
@dspreadbury you might want to chime on this given Dorico's independent modeling of parts, staves and instruments.
